README.md

```diff
@@ -1,207 +1,225 @@
 # SignalFx Java Agent
 
-This is the SignalFx Java Agent, a [JVM
-agent](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
-to automatically instrument your Java application to capture and report
-distributed traces to SignalFx. Enabling the agent is done by downloading the
-[latest JAR](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
-and adding its path to your JVM startup options:
+The SignalFx Java Agent is a
+[Java Virtual Machine](https://docs.oracle.com/javase/7/docs/api/java/lang/instrument/package-summary.html)
+(JVM) agent that automatically instruments your Java application to capture and
+report distributed traces to SignalFx. Download the JAR for the agent's 
+[latest release](https://github.com/signalfx/signalfx-java-tracing/releases/latest)
+and add its path to your JVM startup options:
 
 ```bash
-$ # Specify the Java Agent with the appropriate option and launch your application as usual
-$ java -javaagent:signalfx-tracing.jar -jar app.jar
+$ java -javaagent:path/to/signalfx-tracing.jar -jar app.jar
```

Comment:
Mind adding
`curl -L https://github.com/signalfx/signalfx-java-tracing/releases/latest/download/signalfx-tracing.jar -o signalfx-tracing.jar` before the java invocation?

Reply:
`-javaagent:./signalfx-tracing.jar` since it's downloaded in the previous step.